### DISCUSSION TITLE
Security Audit findings

### DISCUSSION THREAD

**Discussion by extern-c** [goharbor/harbor#21074] [Q&A] [answered]
Hi.

I am new to Harbor and would like to know if the vulnerabilities identified as part of the 2019 security audit were addressed. More specifically, were the CSRF and SQL injection issues resolved?

Thank you!

**Comment by Vad1mo** [comment, marked as answer]
please take a look here https://github.com/goharbor/harbor/security/advisories,

**Comment by stonezdj** [comment]
All these issues are fixed.

**Reply by extern-c** [reply]
That’s great. Thanks @Vad1mo and @stonezdj.
